pg-core/src/java/org/pg/error/PGErrorIO.java

```diff
@@ -0,0 +1,17 @@
+package org.pg.error;
```

Comment:
@jgdavey Sorry for letting you wait for such long. The PR looks good, and I made a small change. Briefly, I'd like `PGError` class to be the root of all `PG...` exceptions. Second, I renamed `PGIOException` to `PGErrorIO`, because it follows the pattern: `PGError<Subdomain>`, for example `PGErrorResponse` = `PGError` + `Response`.

Reply:
Tests pass, so if you're OK with the changes, I will merge it.

Reply:
Great! Thank you!